proofreader/runner.py

```diff
@@ -0,0 +1,61 @@
+"""Module `runner`
+
+This module
+
+- checks for local config overrides
+- runs `flake8` and `pylint` on the
+- outputs run_command progress to stdout and stderr
+- returns exit code based on the success or failure of both commands
+
+"""
+from __future__ import print_function
+from subprocess import Popen
+import sys
+
+from proofreader.config.utils import (
+    get_flake8_options, get_pylint_options
+)
+
+
+def run(targets, config_dir='.'):
+    # type: (List[str], str) -> None
```

Comment:
python doesn't enforce it yet, but it doesn't error out if you include type information in your function signatures. unevaluated type info is better than a comment.

Reply:
"but it doesn't error out if you include type information in your function signatures." in Python 2 it does.

This is the recommended way to include type annotations in Python 2 supported projects. This can be moved into the signature once Python 2 support for the project is dropped.